receiver/statsdreceiver/protocol/statsd_parser_test.go

```diff
@@ -20,8 +20,12 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/exponential/mapping/logarithm"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/correctnesstests/metrics"
```

Comment:
I don't think we should use testbed here: testbed is an environment for end to end tests, not unit tests. If you need this functionality, we should move it to a shared internal module.

Reply:
Are there similar utilities elsewhere I should find and use, instead? It's quite difficult to write a good test without helpers of this nature to print what's actually not matching correctly. Alternatively, is there an explainer for how I can properly use this code to test the receiver here? This library looks perfect for my use-case.

Reply:
@djaglowski has done some work on testbed so can probably help us here. IMO we should move this diff function to an internal shared module, which shouldn't be a lot of work

Reply:
@djaglowski how would you like to see the testbed code factored? The PR description explains what I did:
> Extended the testbed framework for exponential histogram testing; fixed a TODO in that file -- metric attributes were not being verified and now are.

Reply:
Could this be moved to `internal/coreinternal/emtricstestutil`? Ideally in a separate PR that we can get merged pretty quickly

Reply:
I think so. This means some refactoring because the utilities here (which I found nice, and useful) are somehow tied with a Prometheus dependency, whereas all I wanted was the small utilities. I'll try.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13428